plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/util/PageListBuilder.java

```diff
@@ -94,11 +94,31 @@ public void appendInteger(int value)
         INTEGER.writeLong(nextColumn(), value);
     }
 
+    public void appendInteger(Integer value)
```

Comment:
We could add a helper method likes `SnapshotsTable#checkNonNull` to avoid repeated null check.

Reply:
Updated, also updated related methods in `SnapshotsTable`. Let me know if this is what you imagine